src/Core/src/Platform/iOS/KeyboardAutoManagerScroll.cs

```diff
@@ -316,7 +316,7 @@ internal static void AdjustPosition()
 		nfloat statusBarHeight;
 		nfloat navigationBarAreaHeight;
 
-		if (ContainerView.GetNavigationController() is UINavigationController navigationController)
+		if (View.FindResponder<UINavigationController>() is UINavigationController navigationController)
```

Comment:
Small change in my scrolling code here to use the current UINavigationController instead of the top-most one in case there are multiple UINavigationControllers. This presented an issue from building the UITest for this PR!